releasenotes/notes/acquire-single-channel-ea83cef8d991f945.yaml

```diff
@@ -0,0 +1,24 @@
+---
+prelude: >
+    Acquire and AcquireInstruction now can be applied to a single channel
+features:
+  - |
+    Within the terra API allow acquires be applied to a single qubit.
+    This makes pulse programming more consistent and easier to reason
+    about as all operations in pulse would then be for single channels.
+    For example:
+
+      acquire = Acquire(duration=10)
+      schedule = Schedule()
+      schedule.insert(60, acquire(AcquireChannel(0), MemorySlot(0), RegisterSlot(0)))
+      schedule.insert(60, acquire(AcquireChannel(1), MemorySlot(1), RegisterSlot(1)))
+
+issues:
```

Comment:
This section is usually for unresolved known issues, so I think can be removed here.